backends/advanced/src/advanced_omi_backend/app_config.py

```
@@ -68,11 +63,14 @@ def __init__(self):
         self.qdrant_base_url = os.getenv("QDRANT_BASE_URL", "qdrant")
         self.qdrant_port = os.getenv("QDRANT_PORT", "6333")
-        self.memory_provider = os.getenv("MEMORY_PROVIDER", "chronicle").lower()
-        # Map legacy provider names to current names
-        if self.memory_provider in ("friend-lite", "friend_lite"):
-            logger.debug(f"Mapping legacy provider '{self.memory_provider}' to 'chronicle'")
-            self.memory_provider = "chronicle"
+        # Memory provider from registry
+        try:
+            from advanced_omi_backend.model_registry import get_models_registry
+            _reg = get_models_registry()
+        except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)